### ISSUE TITLE
Middle click with drag no longer works in 3.0.1 [bug]

### ISSUE THREAD

**Issue by lem503** [noah-nuebling/mac-mouse-fix#815]
## Description

Just noticed that my settings were lost and checked in the app what was going on.
I guess in updating to v3 settings did not persist.

Now I noticed that the middle button no longer works as expected. Before I was able to middle-click and drag the screen (e.g. to easily navigate around a whiteboarding app like Miro).

The middle button now only can click. Click and dragging the mouse does nothing.
I would love to be able to click and drag again. This used to work in v2.

## Steps to reproduce

–

## Context

Mac Mouse Fix Version: 3.0.1
macOS Version: 14.3
Mouse Model: Razer Orochi v2

## Additional Comments

This is a major reason for having gotten Mac Mouse Fix and I will try to downgrade to v2 and if that's not possible look for alternatives, which would be sad seeing that I paid for the app.

---

### Console Logs

–

### Crash Reports

–

**Comment by noah-nuebling** [comment]
Hi there, @lem503, thanks for your feedback. You can restore the normal middle-click-and-drag functionality by removing all actions for the 'Middle Button' in Mac Mouse Fix.

You should also be able to navigate around a whiteboarding app like Miro by setting up the 'Click and Drag' to 'Scroll & Navigate' feature for one of your buttons. 

If you would like a refund, please shoot me an email.

I hope that helps! Let me know if you have any more questions.

**Comment by noah-nuebling** [comment]
Oh by the way, I don’t think downgrading to MMF 2 makes sense in your case. I think it’s just a matter of your setup.

**Comment by lem503** [comment]
Hey thanks for getting back @noah-nuebling. I figured out what I was missing.

I didn't get that I had to click and drag in the big + field with my middle mouse button to be able to set the click and drag setting for that button.

I was expecting to see all middle mouse button related settings when I clicked with the middle button on the big +.
I therefore thought that the click + drag feature was no longer there when only the middle click setting showed up.
All good now, thanks for getting back to me (and I'll stay with v3 😄)
